frontend/components/TodayInHistory.tsx

```diff
@@ -30,37 +29,32 @@ const TodayInHistory: FC = () => {
   }, [loading]);
 
   return (
-    <>
-      <div style={{ textAlign: "center" }}>
-        <Typography variant="h6" gutterBottom component="div" fontWeight="bold">
-          Today in History
-        </Typography>
-      </div>
-      <Grid container spacing={1} justifyContent="center" alignItems="center">
-        <>
-          {items.length ? (
-            items.map((module, index) => (
-              <Grid item xs={12} sm={6} md={4} lg={3} xl={3} key={index}>
-                <Link href={module.absoluteUrl}>
-                  <a>
-                    <ModuleUnionCard module={module} />
-                  </a>
-                </Link>
-              </Grid>
-            ))
-          ) : loading ? (
-            <Card>
+    <Grid container spacing={1} justifyContent="center" alignItems="center">
+      <>
+        {items.length ? (
+          items.map((module, index) => (
+            <Grid item key={index}>
+              <Link href={module.absoluteUrl}>
+                <a>
+                  <ModuleUnionCard module={module} />
+                </a>
+              </Link>
```

Comment:
Not related to this PR, but we use the pattern enough that we should just always render `Link` inside `ModuleCard`, and maybe add a prop if we need to disable it.